$NetBSD: patch-src_Makefile,v 1.4 2015/07/13 15:03:22 thomasklausner Exp $

Build download(1) with Gtk+ 3.0 for X11
Build both download(1) and surfer(1) with WebKit for Gtk+ 3.0

--- src/Makefile.orig	2015-01-26 22:56:38.000000000 +0000
+++ src/Makefile
@@ -23,15 +23,15 @@ subdirs:
 	@for i in $(SUBDIRS); do (cd "$$i" && $(MAKE)) || exit; done
 
 download_OBJS = $(OBJDIR)download.o
-download_CFLAGS = $(CPPFLAGSF) $(CPPFLAGS) -D WITH_WEBKIT $(CFLAGSF) $(CFLAGS) `pkg-config --cflags libSystem gtk+-2.0 webkit-1.0`
-download_LDFLAGS = $(LDFLAGSF) $(LDFLAGS) `pkg-config --libs libSystem gtk+-2.0 webkit-1.0` -lintl
+download_CFLAGS = $(CPPFLAGSF) $(CPPFLAGS) -D WITH_WEBKIT $(CFLAGSF) $(CFLAGS) `pkg-config --cflags libSystem gtk+-x11-3.0 webkitgtk-3.0`
+download_LDFLAGS = $(LDFLAGSF) $(LDFLAGS) `pkg-config --libs libSystem gtk+-x11-3.0 webkitgtk-3.0` -lintl
 
 $(OBJDIR)download: $(download_OBJS)
 	$(CC) -o $(OBJDIR)download $(download_OBJS) $(download_LDFLAGS)
 
 surfer_OBJS = $(OBJDIR)surfer.o $(OBJDIR)ghtml.o $(OBJDIR)callbacks.o $(OBJDIR)main.o
-surfer_CFLAGS = $(CPPFLAGSF) $(CPPFLAGS) -D WITH_WEBKIT $(CFLAGSF) $(CFLAGS) `pkg-config --cflags libDesktop webkit-1.0`
-surfer_LDFLAGS = $(LDFLAGSF) $(LDFLAGS) `pkg-config --libs libDesktop webkit-1.0` -lintl
+surfer_CFLAGS = $(CPPFLAGSF) $(CPPFLAGS) -D WITH_WEBKIT $(CFLAGSF) $(CFLAGS) `pkg-config --cflags libDesktop webkitgtk-3.0`
+surfer_LDFLAGS = $(LDFLAGSF) $(LDFLAGS) `pkg-config --libs libDesktop webkitgtk-3.0` -lintl
 
 $(OBJDIR)surfer: $(surfer_OBJS)
 	$(CC) -o $(OBJDIR)surfer $(surfer_OBJS) $(surfer_LDFLAGS)
